content/docs/user-guide/project-config/project-manager.md

```diff
@@ -0,0 +1,151 @@
+
+## Launching Project Manager
+
+To launch Project Manager, do one of the following:
+
+* Open the **Project Manager** shortcut that was placed on your desktop during installation.
+* Launch `o3de.exe` from the engine or project build directory.
+
+The location of `o3de.exe` depends on how the engine binaries were built:
+
+{{< tabs name="o3de.exe location" >}}
+{{% tab name="Source engine" %}}
+
+When you build a source engine, you specify a build directory, such as `build/windows_vs2019` or `build/linux`. The binaries are located in a subdirectory of the build directory. The path name of this subdirectory depends on the build configuration you chose. Example: `bin/profile`.
+
+To launch Project Manager, do the following:
+
+1. Open a file browser or command line window on your computer and navigate to the O3DE engine or project directory that contains your build directory.
+
+1. Locate and launch the Project Manager application, `o3de.exe`, from `<BUILD_DIRECTORY>/bin/<BUILD_CONFIGURATION>`.
+
+Example:
+
+```cmd
+build\windows_vs2019\bin\profile\o3de.exe
+```
+
+{{% /tab %}}
+{{% tab name="Installed or pre-built SDK engine" %}}
+
+To launch Project Manager, do the following:
+
+1. Open a file browser or command line window on your computer and navigate to your O3DE install directory.
+
+1. Locate and launch the Project Manager application, `o3de.exe`, from `<INSTALL_DIRECTORY>/bin/<PLATFORM>/<BUILD_CONFIGURATION>`.
+
+Example:
+
+```cmd
+bin\Windows\profile\o3de.exe
+```
+
+{{< important >}}
+If you built the engine from source using the `INSTALL` target, make sure that you launch Project Manager from the installed engine's build directory, _not_ the engine's build directory. This is important because the engine that Project Manager uses for various operations (creating new projects, for example) is determined based on the location of the `o3de.exe` application.
+{{< /important >}}
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Project Manager reference
+
+The Project Manager application contains the following screens:
+
+* Projects
+* Engine
+* Project details
+* Project settings
+* Configure gems
+
+### Projects
+
+The **Projects** tab is the home screen for Project Manager. Here you will find all of your registered O3DE projects.
+
+![Projects tab with legend](/images/user-guide/project-config/project-manager/projects-tab-legend.png)
+
+When one or more projects are registered on your computer, the **New Project...** menu **(1)** appears in the upper right corner of the window, containing the following choices:
+
+| Action | Description |
+| - | - |
+| **Create New Project** | Starts the new project workflow. |
+| **Add Existing Project** | Opens a "Select Project Directory" browse dialog box, from which you can add an existing O3DE project to Project Manager. This will also register the project in your O3DE manifest. |
```

Comment:
```suggestion
| **Add Existing Project** | Opens a **Select Project Directory** browse dialog box, from which you can add an existing O3DE project to Project Manager. This will also register the project in your O3DE manifest. |
```